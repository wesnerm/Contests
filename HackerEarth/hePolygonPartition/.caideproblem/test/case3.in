40
159 729
667 327
5 640
673 451
420 333
298 402
684 549
302 99
660 65
1000 543
627 625
807 350
364 139
619 743
777 917
529 371
356 706
318 418
152 319
168 661
537 413
459 302
4 467
75 754
58 405
118 10
177 443
694 535
923 317
963 866
20 930
674 504
804 928
46 979
231 884
948 340
897 339
5 214
166 504
345 898