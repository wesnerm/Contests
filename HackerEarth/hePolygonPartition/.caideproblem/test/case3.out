11
1 18
1 21
4 38 36 33 34
4 31 26 29 30
4 19 12 14 1
4 27 13 28 17
4 39 5 2 7
4 3 22 4 11
4 20 6 16 32
5 23 9 10 40 24
5 25 8 37 15 35
